src/components/organisms/Footer/data.ts

```diff
@@ -14,40 +15,52 @@ export const formatter = (data: GraphQLFooterFields): FooterData => ({
 		social: { items: menuItemData.formatter(data.social?.nodes ?? []) },
 	},
 	siteTitle: data.generalSettings?.title ?? 'Superstack',
-	isHome: data.node?.isFrontPage ?? false,
 });
 
-export const getData = async (uri: string): Promise<FooterData> => {
+export const getData = async (language: Locale): Promise<FooterData> => {
 	const query = gql`
-		query footerQuery($uri: String!) {
-			footer: menuItems(where: { location: FOOTER }, first: 9999) {
+		query footerQuery {
+			footer: menuItems(
+				where: {
+					location: FOOTER
+					${configs.isMultilang ? `, language: ${language.toUpperCase()}` : ''}
+				}
+				first: 9999
+			) {
 				nodes {
 					...menuItemFragment
 				}
 			}
-			legal: menuItems(where: { location: LEGAL }, first: 9999) {
+			legal: menuItems(
+				where: {
+					location: LEGAL
+					${configs.isMultilang ? `, language: ${language.toUpperCase()}` : ''}
+				}
+				first: 9999
+			) {
 				nodes {
 					...menuItemFragment
 				}
 			}
-			social: menuItems(where: { location: SOCIAL }, first: 9999) {
+			social: menuItems(
+				where: {
+					location: SOCIAL
+					${configs.isMultilang ? `, language: ${language.toUpperCase()}` : ''}
+				}
+				first: 9999
+			) {
 				nodes {
 					...menuItemFragment
 				}
 			}
 			generalSettings {
 				title
 			}
-			node: nodeByUri(uri: $uri) {
-				... on Page {
-					isFrontPage
-				}
-			}
 		}
 		${menuItemData.fragment}
 	`;
 
-	const data = await fetchAPI(query, { variables: { uri } });
+	const data = await fetchAPI(query, { variables: {} });
```

Comment:
Here you can completely remove the second parameter

```suggestion
	const data = await fetchAPI(query);
```

Reply:
Done !